docs/source/release_notes/release-notes.rst

```diff
@@ -61,6 +61,10 @@ develop
          - AtlasDB now depends on Tritium 0.8.1, allowing products to upgrade Tritium without running into ``NoClassDefFound`` errors.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2639>`__)
 
+    *    - |improved|
+         - AtlasDB now depends on Tritium 0.8.2, allowing products to upgrade Tritium without running into ``NoSuchFieldError`` errors.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/2641>`__)
```

Comment:
this and the above release note should be merged in some way, but we can clean this up when sorting out the release

Reply:
tweaked and updated to 0.8.3